imports/plugins/core/accounts/client/containers/auth.js

```diff
@@ -153,12 +149,26 @@ class AuthContainer extends Component {
         });
       }
     });
-  }
+  };
 
-  hasPasswordService = () => !!Package["accounts-password"]
+  hasPasswordService = () => !!Package["accounts-password"];
 
   renderAuthView() {
     if (this.props.currentView === "loginFormSignInView") {
+      if (this.props.currentRoute.route.name === "reset-password") {
```

Comment:
@aldeed is this where & how I should be checking if we need to display the`UpdatePassword` form?

Reply:
@nnnnat In the `registerPackage` call for the accounts plugin, I see this:

```
{
    route: "/reset-password/:token/:status?",
    template: "loginFormUpdatePassword",
    workflow: "none",
    meta: { noAdminControls: true },
    name: "reset-password",
    label: "reset-password"
  }, {
    route: `/${ENROLL_URI_BASE}/:token/:status?`,
    template: "loginFormUpdatePassword",
    workflow: "none",
    meta: { noAdminControls: true },
    name: ENROLL_URI_BASE,
    label: "Account Enroll"
  }
```

These may need to get updated and moved into the hydra-oauth plugin, but I'm not sure the exact solution without playing around with it a lot more. @impactmass may have a better idea of what to do. But essentially we need those routes to work again, and make sure the email is linking to the `reset-password` route.

Generally the way you're doing this here feels a little bit off. It seems like you could do this in the `Login` component where it's already examining the `currentRoute`, and then set a new value for `currentView`, which you can check here.

Reply:
@nnnnat, sorry Nat, I need to look a bit more to understand why this isn't working anymore and then be able to suggest approaches.  

My recollection is that those routes pasted above were loading up fine, and I remember there is a use of that props.currentRoute somewhere in the signup/in components to toggle the views. But, I'll need to check